Makefile.in

```diff
@@ -318,13 +318,20 @@ distprep: catman-do
 	$(AUTORECONF)
 	-rm -rf autom4te.cache
 
-install: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-sysconf host-key check-config
+install: $(CONFIGFILES) $(MANPAGES) $(TARGETS) install-files install-iron-files install-sysconf host-key check-config
```

Comment:
Maybe we should do `install-iron-files` instead of `install-files`, rather than in addition? That will probably host the test scripts, so maybe that needs to get sorted out before we can make this change.